src/com/clancraft/turnmanager/TMConstants.java

```diff
@@ -30,24 +34,47 @@
             + "%s and %s have been swapped in the turn sequence.";
     public static final String SWAP_PLAYER_FAILED = PLUGIN_PREFIX + ChatColor.RED
             + "%s and %s could not be swapped in the turn sequence.";
+    public static final String SWAP_PLAYER_MISSING_ARGUMENT_ERROR = PLUGIN_PREFIX + ChatColor.RED
+            + "/tm swap requires two arguments!\n" + PLUGIN_PREFIX + "/tm swap [player1] [player2]";
 
     public static final String ANNOUNCE_CURRENT_PLAYER = PLUGIN_PREFIX + "Current player is: %s";
     public static final String ANNOUNCE_SEQUENCE = PLUGIN_PREFIX + "Sequence: %s";
 
-    public static final String TIMER_INITIAL = "Turn has started.";
-    public static final String TIMER_COUNTDOWN = "%d minutes remaining in this turn.";
-    public static final String TIMER_TIMEUP = "The time for this turn is up!";
-    public static final String TIMER_OVERTIME = "This turn is %d minutes over time!";
+    public static final String TURN_MISSING_ARGUMENT_ERROR = PLUGIN_PREFIX + ChatColor.RED
+            + "/tm turn requires an argument!\n" + PLUGIN_PREFIX + "/tm turn [argument]";
 
-    //TODO change SHIELD_PERIMETER_BREACH, SHIELD_TELEPORT_BREACH, SHIELD_NO_ACTIVE_PLAYER and message
-    public static final String SHIELD_PERIMETER_BREACH = "You have entered a shielded area. Your location has been reverted a the previously recorded location.";
-    public static final String SHIELD_TELEPORT_BREACH = "Unable to teleport. You are shielded from the current player's turn.";
-    public static final String SHIELD_NO_ACTIVE_PLAYER = "Unable to teleport. There is no active player having a turn.";
+    public static final String CYCLE_MISSING_ARGUMENT_ERROR = PLUGIN_PREFIX + ChatColor.RED
+            + "/tm cycle requires an argument!\n" + PLUGIN_PREFIX + "/tm cycle [argument]";
 
-    //TODO change DATE_SYNC_ERROR, DATE_REGISTER_DUPLICATE, and DATE_UNREGISTER_MISSING message
-    public static final String DATE_SYNC_ERROR = "%s's date is synced to the world date. Unable to change date.";
-    public static final String DATE_REGISTER_DUPLICATE = "%s is already register";
-    public static final String DATE_UNREGISTER_MISSING = "Unable to unregister %s, player not found";
+    public static final String TIMER_MISSING_ARGUMENT_ERROR = PLUGIN_PREFIX + ChatColor.RED
+        + "/tm timer requires an argument!\n" + PLUGIN_PREFIX + "/tm timer [argument]";

```

Comment:
nitpick but the others have 8 spaces instead of 4, with 8 signifying it's part of the same statement and not an indent

Reply:
Hm I'll check it out

Reply:
Oh wait it _is_ part of the same statement. I went to the next line for added readability since it's quite long.

Reply:
Oh wait now I see it after seeing all the other changes. Minor but consistency is important so I'll change it before merging